### ISSUE TITLE
Error when I try generate a model

### ISSUE THREAD

**Issue by nailtonvieira** [krlove/eloquent-model-generator#10]
php artisan krlove:generate:model GameRound --table-name=gameRound

[Symfony\Component\Debug\Exception\FatalThrowableError]  
  Call to a member function getColumns() on null

```php

<?php

use Illuminate\Database\Schema\Blueprint;
use Illuminate\Database\Migrations\Migration;

class CreateGameRoundTable extends Migration
{
    /**
     * Run the migrations.
     * @table gameRound
     *
     * @return void
     */
    public function up()
    {
        Schema::create('gameRound', function (Blueprint $table) {
            $table->integer('idgameRound')->unsigned();
            $table->integer('question_idquestion1')->unsigned();
            $table->integer('question_idquestion2')->unsigned();
            $table->integer('question_idquestion3')->unsigned();
            $table->integer('question_idquestion4')->unsigned();
            $table->dateTime('startDate');
            $table->dateTime('endDate')->nullable();
            $table->integer('user_idUser')->unsigned();
            $table->string('question1Answer', 45)->nullable();
            $table->string('question2Answer', 45)->nullable();
            $table->string('question3Answer', 45)->nullable();
            $table->string('question4Answer', 45)->nullable();
            $table->integer('round');
            $table->integer('game_idgame')->unsigned();
            # Indexes
            $table->index('question_idquestion1');
            $table->index('question_idquestion2');
            $table->index('question_idquestion3');
            $table->index('question_idquestion4');
            $table->index('user_idUser');
            $table->index('game_idgame');


            $table->foreign('question_idquestion1', 'fk_gameRound_question1_idx')
                ->references('idquestion')->on('question')
                ->onDelete('no action')
                ->onUpdate('no action');

            $table->foreign('question_idquestion2', 'fk_gameRound_question2_idx')
                ->references('idquestion')->on('question')
                ->onDelete('no action')
                ->onUpdate('no action');

            $table->foreign('question_idquestion3', 'fk_gameRound_question3_idx')
                ->references('idquestion')->on('question')
                ->onDelete('no action')
                ->onUpdate('no action');

            $table->foreign('question_idquestion4', 'fk_gameRound_question4_idx')
                ->references('idquestion')->on('question')
                ->onDelete('no action')
                ->onUpdate('no action');

            $table->foreign('user_idUser', 'fk_gameRound_user1_idx')
                ->references('idUser')->on('user')
                ->onDelete('no action')
                ->onUpdate('no action');

            $table->foreign('game_idgame', 'fk_gameRound_game1_idx')
                ->references('idgame')->on('game')
                ->onDelete('no action')
                ->onUpdate('no action');
        });
    }

    /**
     * Reverse the migrations.
     *
     * @return void
     */
     public function down()
     {

        Schema::drop('gameRound');
    }
}

```
Thanks!

**Comment by krlove** [comment]
Hi @nailtonvieira ,

Thanks for your contribution. I've applied fix (version 1.2.1), can you check it out?

**Comment by krlove** [comment]
Closing the issue, you can reopen it if the problem still exists.
